tests/run-tests.py

```diff
@@ -588,16 +617,29 @@ def run_tests(pyb, tests, args, result_dir):
             print("FAIL ", test_file)
             failed_tests.append(test_name)
 
-        test_count += 1
+        test_count.increment()
 
     if args.list_tests:
```

Comment:
This `--list-tests` option no longer works.  To make it work again, if this option is given, then set `num_threads = 1`.